src/main/resources/beam-template.conf

```diff
@@ -764,6 +764,9 @@ beam.debug {
 
     writeRealizedModeChoiceFile = "boolean | false"
     messageLogging = "boolean | false"
+    # the max of the next 2 values is taken for the initialization step
+    simulationStuckThresholdMin = "int | 999999"
```

Comment:
After reading the implementation I propose a rename to something like `maxSimulationStepTimeBeforeConsideredStuck` and `maxInitializationTimeBeforeConsideredStuck`.